testify/test_runner.py

```diff
@@ -31,6 +30,10 @@
 __author__ = "Oliver Nicholas <bigo@yelp.com>"
 __testify = 1
 
+EX_OK = 0  # successful termination
+EX_DATAERR = 65  # EX_DATAERR -- The input data was incorrect in some way.
+EX_SOFTWARE = 70  # EX_SOFTWARE -- An internal software error has been detected.
```

Comment:
also, maybe we should pick different numbers to be able to differentiate between unexpected traceback (I think exit code 1?) -- perhaps p3 though